### PR TITLE
Minor Changes

### PR DESCRIPTION
**Pull request by tcrms**

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ export async function DiscordRequest(endpoint, options) {
     headers: {
       Authorization: `Bot ${process.env.DISCORD_TOKEN}`,
       'Content-Type': 'application/json; charset=UTF-8',
+      'User-Agent': 'Mozilla/5.0 (compatible; Discordbot/2.0; +https://discordapp.com)',
```
Comment (splatterxl) [inline utils.js:28]: This is invalid. The user agent must be `Discordbot ($url, $version)` as per the docs.

**Review by splatterxl — CHANGES_REQUESTED** [review]

File: utils.js (hunk 1 above)
Comment (tcrms) [reply utils.js:28]: mb

Comment (tcrms) [reply utils.js:28]: do you have a 'example' from $version?

File: utils.js [hunk 2]
```diff
@@ -17,14 +17,15 @@ export function VerifyDiscordRequest(clientKey) {
...
-  const url = 'https://discord.com/api/v9/' + endpoint;
+  const url = 'https://discord.com/api/v10/' + endpoint;
   // Stringify payloads
   if (options.body) options.body = JSON.stringify(options.body);
   // Use node-fetch to make requests
   const res = await fetch(url, {
     headers: {
       Authorization: `Bot ${process.env.DISCORD_TOKEN}`,
       'Content-Type': 'application/json; charset=UTF-8',
+      'User-Agent': 'DiscordBot (https://github.com/discord/discord-example-app, null)',
```
Comment (splatterxl) [inline utils.js:28]: I'm not in favour of using null here
```suggestion
      'User-Agent': 'DiscordBot (https://github.com/discord/discord-example-app, 0.0.0)',
```

**Review by splatterxl — APPROVED** [review]

**Pull request merged**